ui/types/vault/router.d.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
```

Comment:
I followed these [docs](https://docs.ember-cli-typescript.com/cookbook/working-with-route-models) to add this file which returns the resolved route model - but I wasn't sure if this was the most intuitive place to put these exports?